pkg/controller/deployment/deployment_controller.go

```diff
@@ -56,9 +59,14 @@ const (
 	MaxRetries = 5
 )
 
+func getDeploymentControllerKind() unversioned.GroupVersionKind {
```

Comment:
This is an unnecessary indirection in the code and the deployment controller isn't the easiest one to maintain so the simpler things are the better.